corehq/apps/enterprise/forms.py

```diff
@@ -30,14 +32,88 @@ class EnterpriseSettingsForm(forms.Form):
+        required=False,
+        initial=ExportFileType.EXCEL_2007_PLUS,
+        choices=ExportFileType.CHOICES,
+    )
+
+    forms_auto_convert = forms.BooleanField(
+        label=ugettext_lazy("Automatically convert dates and multimedia links for Excel"),
+        required=False,
+        help_text=ugettext_lazy("Leaving this checked will ensure dates appear in excel format. "
+                                "Otherwise they will appear as a normal text format. This also allows "
+                                "for hyperlinks to the multimedia captured by your form submission.")
+    )
+
+    forms_auto_format_cells = forms.BooleanField(
+        label=ugettext_lazy("Automatically format cells for Excel 2007+"),
+        required=False,
+        help_text=ugettext_lazy("If this setting is not selected, your export will be in Excel's generic format. "
+                                "If you enable this setting, Excel will format dates, integers, decimals, "
+                                "boolean values (True/False), and currencies.")
+    )
+
+    forms_include_duplicates = forms.BooleanField(
+        label=ugettext_lazy("Include duplicates and other unprocessed forms"),
+        required=False,
+    )
+
+    forms_expand_checkbox = forms.BooleanField(
+        label=ugettext_lazy("Expand checkbox questions"),
+        required=False,
+    )
+
+    cases_filetype = forms.ChoiceField(
+        label=ugettext_lazy("Default File Type"),
+        required=False,
+        initial=ExportFileType.EXCEL_2007_PLUS,
+        choices=ExportFileType.CHOICES,
+    )
+
+    cases_auto_convert = forms.BooleanField(
+        label=ugettext_lazy("Automatically convert dates and multimedia links for Excel"),
+        required=False,
+        help_text=ugettext_lazy("Leaving this checked will ensure dates appear in excel format. "
+                                "Otherwise they will appear as a normal text format. This also allows "
+                                "for hyperlinks to the multimedia captured by your form submission.")
+    )
+
+    odata_include_duplicates = forms.BooleanField(
+        label=ugettext_lazy("Include duplicates and other unprocessed forms"),
+        required=False,
+    )
+
+    odata_expand_checkbox = forms.BooleanField(
+        label=ugettext_lazy("Expand checkbox questions"),
+        required=False,
+    )
+
     def __init__(self, *args, **kwargs):
         self.domain = kwargs.pop('domain', None)
         self.account = kwargs.pop('account', None)
+        self.export_settings = kwargs.pop('export_settings', None)
         kwargs['initial'] = {
             "restrict_domain_creation": self.account.restrict_domain_creation,
             "restrict_signup": self.account.restrict_signup,
             "restrict_signup_message": self.account.restrict_signup_message,
         }
+
+        if DEFAULT_EXPORT_SETTINGS.enabled(self.domain):
+            export_settings_args = {
```

Comment:
Could this be replaced by `dict(self.export_settings)`?

Reply:
I created an `as_dict()` method on `DefaultExportSettings`. I did this instead of implementing the `__dict__` method because I wanted to omit the `account` attribute, and this seemed the most explicit way to provide a dictionary of a class where some attributes are not included.